doc/rpc-spec.md

```diff
@@ -22,9 +22,109 @@ ALl RPC calls in Pocket Core will be HTTP `POST`
 ### Client Namespace
 The default namespace contains functions that are pertinent to the execution of the Pocket Node.
 
-// todo
+- /v1/client/dispatch
+> sends a dispatch request to the network and get the nodes that will be servicing your requests for the session.
```

Comment:
This spec has very minimal information on how to execute .

I think we need examples, the actual structure.

See: https://github.com/ethereum/wiki/wiki/JSON-RPC#web3_clientversion

For example

Reply:
I don't see any standards being set with this doc. 
Remember that client side and future rpc development should conform to this spec. But this isn't really much of a spec

Reply:
Also, you may want to look into #696 before finishing the spec as it affects your work

Reply:
After call with @oten91 it seems as if this commit was on the wrong branch, and does not satisfy #503